stock_vertical_lift/models/vertical_lift_operation_base.py

```diff
@@ -11,6 +11,90 @@
 _logger = logging.getLogger(__name__)
 
 
+# The following methods have been copied from 'shopfloor' module (OCA/wms)
+# TODO: we should move them in a generic module
+
+
+def split_other_move_lines(move, move_lines, intersection=False):
```

Comment:
As we discussed on the other one, `intersection` is an issue, it doesn't compute the correct `qty_to_split` and should be removed (in favor of filtering the lines before the call to `split_other_move_lines`. Shouldn't you remove this parameter here?

Reply:
I could remove it here yes as it is unused.
But regarding the comment of @simahawk regarding this parameter, I checked and - I could be wrong - but it was difficult to drop it...

Reply:
IIRC we could make the intersection before and pass these move lines

Reply:
Yep but I was struggling with the computation of the qty to split which differs with/without intersection (and I could not understand anymore why there is this difference). Need to take time to analyse again this stuff.